### PR TITLE
Move GlowSignInButton to glow-react

### PR DESCRIPTION
**Pull request by MeLlamoPablo**
This PR moves `GlowSignInButton` from `example-next-js` to `@glow-app/glow-react`.

Devs will need to manually import the css:

```typescript
import "@glow-app/glow-react/dist/styles.css";
```

### CODE REVIEW

File: packages/glow-react/package.json [hunk 1]
```diff
@@ -9,19 +9,25 @@
     "src/**/*"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "npm run build:js && npm run build:css",
+    "build:js": "tsc",
+    "build:css": "sass src/styles/index.scss:dist/styles.css",
     "release": "tsc && yarn version && npm publish --access public"
```
Comment (vpontis) [inline packages/glow-react/package.json]: I think we need to build both js and css before releasing, right?

**Review by vpontis — APPROVED** [review]

File: packages/glow-react/package.json (hunk 1 above)
Comment (MeLlamoPablo) [reply packages/glow-react/package.json]: Right! Good catch

Comment (MeLlamoPablo) [reply packages/glow-react/package.json]: Also it's interesting that we combine npm and yarn in that script and use pnpm for the rest haha

Have you looked into yarn v2? I used in the past and liked it a lot

Comment (vpontis) [reply packages/glow-react/package.json]: Haha I think we should just use `pnpm` but I don't know what all the commands are for like version / publish.

**Pull request merged**